scripts/g.extension/g.extension.py

```diff
@@ -1296,8 +1296,17 @@ def get_multi_addon_addons_which_install_only_html_man_page():
     )
     if not os.path.exists(addons_paths_file):
         get_addons_paths(gg_addons_base_dir=options["prefix"])
-    with open(addons_paths_file) as f:
-        addons_paths = json.loads(f.read())
+    try:
+        with open(addons_paths_file) as f:
+            addons_paths = json.loads(f.read())
+    except EnvironmentError:
```

Comment:
Why EnvironmentError? Isn't OSError more appropriate here.

Reply:
Yes you are right.